libs/common/src/helpers/date.test.ts

```diff
@@ -3,6 +3,15 @@ import { getNowAsUtcISOString, getMonthName } from './date';
 
 describe('date helpers', () => {
   describe('getNowAsUtcISOString', () => {
+    beforeAll(() => {
+      jest.useFakeTimers();
+      jest.setSystemTime(new Date());
```

Comment:
im not versed in this, but I'd assume this is already set as the current date when you start using fake timers -- so we dont need this line

Reply:
I can totally believe that, was struggling to find hard evidence of that in the docs so was erring on the side of caution but will do another lil investigation

Reply:
I believe it, have removed that line